pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1941,14 +1962,14 @@ class Patch(CommonObject, object):
     frequency : float, None
         The patch frequency, it is used in prediction formulas. If it is None, the patch frequency will be that of the
         layer or of the stackup.
-    patch_width : float
+    dx : float
         The patch width.
     signal_layer : :class:`pyaedt.modeler.stackup_3d.Layer3D`
         The signal layer where the patch will be drawn.
     dielectric_layer : :class:`pyaedt.modeler.stackup_3d.Layer3D`
         The dielectric layer between the patch and the ground layer. Its permittivity and thickness are used in
         prediction formulas.
-    patch_length : float, None, optional
+    dy : float, None, optional
         The patch length. By default, it is None and so the length is calculated by prediction formulas.
     patch_position_x : float, optional
         Patch x position, by default it is 0.
```

Comment:
```suggestion
        Patch x position. The default is ``0``.
```
The subsequent parameter descriptions should also be edited to meet the standard wording.